.wordlist.txt

```diff
@@ -47,3 +47,6 @@ UI
 UUID
 VM
 YAML
+Terraform
+OAuth
+qa
```

Comment:
You can remove this (I mean, `qa`) if you apply the double-ticks suggestion I've made.